File: src/main/java/com/shreyash/BankApplication/DTO/BankResponse.java
package com.shreyash.BankApplication.DTO;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class BankResponse {

    private String responseCode;
    private String responseMessage;
    private AccountInfo accountInfo;
}

File: src/main/java/com/shreyash/BankApplication/dto/BankResponse.kt
package com.shreyash.BankApplication.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
data class BankResponse (

    val responseCode:String,
    val responseMessage:String,
    val accountInfo: AccountInfo,
){


}

